MLnz20/README.md

```diff
@@ -0,0 +1,35 @@
+# MLnz20 
+This file is a _evaluation dataset_ of the NZ Local network magnitude MLnz20 dataset derived from offline processed Geonet Rapid event data and computed over a separate instance.
+
+It is a SeisComp _development_ magnitude implementation derived from _Rhoades et al._ (2021). 
+This computation is originally based on a GNS Seiscomp Nightly V7.0 (Development) implementation and will be upgraded over time. Results are to be considered informational for science and development purposes and subject to modifications.
+
+This dataset will be superseded as MLnz20 is implemented into GeoNet Rapid Production chain.
+
+## Evaluation Dataset description
+MLnz20 network magnitudes are processed routinely offline every month from GeoNet rapid event automatic and manual solutions. More frequent updates are provided as required (event response for instance). Data are extracted from computed seiscompxml event files and parsed into csv files.  
```

Comment:
- change `GeoNet Rapid` to `GeoNet Rapid` for consistency with previous paragraph
- change `(event response for instance)` to `(e.g. in response to large earthquake events)`